hedera-node/docs/maintainers-guide.md

```diff
@@ -62,17 +77,20 @@ branches and branch related operations; therefore, it is critical that the branc
 tagging strategy.
 
 [GitFlow branching model](https://nvie.com/posts/a-successful-git-branching-model/) is elected pattern for the 
-development life cycle.
+development life cycle.  More or less.  (N.B.: Diagram doesn't match the actual process described below.)
 
 <p>
     <img src="./assets/gitflow-branching-model.png"/>
 </p>
 
-Note especially the roles of the `main` and `develop` branches:
+Note especially the roles of the `develop` and `release/0.nn` branches:
 
 - `develop` is the default branch, the target of active development, and should at all times should be a viable candidate
   for the next release.
-- `main` is a tightly-controlled branch that release engineering uses for final tags deployed to production.
+- For each release a branch named `release/0.nn` is made from the current `develop`.  After the branch is cut commits can be made 
+  to it (then cherry-picked back to `develop`).
```

Comment:
We commit to develop and cherry-pick to release.

Reply:
Actually, in many cases we should probably start with the release and then to `develop` for bug fixes which are identified to be affecting one or more releases.

Reply:
That's not how it is being done currently, as best I can tell.  In nearly all cases that I have seen we currently fix on develop, then cherry-pick to release.  Part of that is we sometimes find the issue is already fixed on develop before it shows up in a release; part of it is that fixing on develop first generally avoids unintentionally re-introducing already-fixed bugs.

Reply:
I thought the newest agreed procedure internally was put the patch into the latest.release branch then cherry pick it into all the earlier releases that need it and then finally into develop. Will confirm internally and make sure this reflects it.